domains/transfer/Transfer.API/API.IntegrationTests/DbContextExtension.cs

```diff
@@ -34,12 +34,9 @@ public static async Task<List<T>> RepeatedlyQueryUntilCountIsMet<T>(this DbConte
     public static async Task TruncateTransferAgreementsTables(this ApplicationDbContext dbContext)
```

Comment:
We should delete this entire method. If we have dropped the table, we don't need to truncate it no more.